tests/unit/ext/switches/brocade.py

```diff
@@ -20,77 +20,77 @@
 from haas.ext.obm.ipmi import Ipmi
 
 MODE_RESPONSE_ACCESS = """
```

Comment:
Is there  away to apply `noqa` to large swaths of code? seems like all of these definitions should just be exempt, rather than having to add `# noqa` after each line that overflows.